platforms/Windows/bundle/installer.wixproj

```diff
@@ -20,6 +20,7 @@
     <ProjectReference Include="..\cli\asserts\cli.asserts.wixproj" BindName="cli.asserts" />
     <ProjectReference Include="..\dbg\asserts\dbg.asserts.wixproj" BindName="dbg.asserts" />
     <ProjectReference Include="..\ide\asserts\ide.asserts.wixproj" BindName="ide.asserts" />
+    <ProjectReference Include="..\python\asserts\python.wixproj" BindName="python" />
```

Comment:
```suggestion
    <ProjectReference Include="..\python\asserts\python.asserts.wixproj" BindName="python.asserts" />
```

Reply:
Marking this as resolved as I believe that we are currently planning to go with the shared installation.